cfme/containers/container.py

```diff
@@ -93,6 +44,34 @@ def is_displayed(self):
         return self.summary.is_displayed
 
 
+class ContainerDetailsView(ContainerObjectDetailsBaseView):
+    pass
+
+
+class Container(WidgetasticTaggable, Navigatable):
+
+    PLURAL = 'Containers'
+    all_view = ContainerAllView
```

Comment:
Q: are you sure you need these views here ? I'm asking because those have to be obtained via navigate_to in most cases.

Reply:
Yes in some cases I need the view class of the object without navigate to that page, like here: https://github.com/gshefer/integration_tests/blob/8c4a1d04df01c705a22ef19c3e6740fd1e4ed103/cfme/tests/containers/test_breadcrumbs.py#L50